admin_scripts/systemd_policy_cleanup.json

```diff
@@ -0,0 +1,12 @@
+ {
+  "name": "System - Slet brugermappen ved opstart og nedlukning",
+  "description": "Installerer en system-tjeneste, der sletter alt indhold i publikum-brugerens hjemmemappe ved opstart og nedlukning af systemet.\n\nUdarbejdet af Alexander Faithfull <af@magenta.dk>.",
+  "parameters": [
+    {
+      "name": "Aktivér?",
+      "type": "string",
+      "required": false,
+      "description": "system-tjenesten vil blive fjernet, hvis dette parameter er tomt eller ordet 'falsk'"
```

Comment:
"denne parameter"

Reply:
Corrected, but this mistake survives in a lot of the other `.json` files. (Luckily, they're not actually used by anything yet.)